client/src/Utils/vendor/useContactFields.ts

```diff
@@ -49,36 +51,49 @@ const useContactFields = (contactStatus?: InteractedContact['contactStatus']) =>
             );
             const isLooseContact = checkIsLooseContact(contact.contactType);
 
-            if (emptyFieldNames.length > 0 || isLooseContact)
-                return {valid: false, error: generateErrorMessage(emptyFieldNames, isLooseContact, validationReason)};
+            const isStatusCompleted = contact.contactStatus === COMPLETE_CONTACT_QUESTIONING_STATUS;
+
+            if ((validationReason === ValidationReason.EXCEL_LOADING && isStatusCompleted) ||
+                 emptyFieldNames.length > 0 || isLooseContact)
+                return {valid: false, error: generateErrorMessage(emptyFieldNames, isLooseContact, validationReason, isStatusCompleted)};
             else return {valid: true};
         }
     };
 
-    const buildEmptyFieldsString = (emptyFields: string[], isLooseContact: boolean) : string => {
+    const buildInValidFieldsString = (emptyFields: string[], isLooseContact: boolean) : string => {
         let message = '';
         if(emptyFields.length > 0) {
             const fieldWord = `לא מילאת את ${emptyFields.length > 1 ? 'שדות ' : 'שדה '}`;
             const emptyFieldProblem = emptyFields.map(fieldName => ContactedPersonFieldMapper[fieldName as keyof typeof ContactedPersonFieldMapper]);
             message = fieldWord.concat(emptyFieldProblem.join(', '));
             if(isLooseContact) message = message.concat(' ו')
         }
-
         if(isLooseContact) {
             const looseContactErrorMessage = 'סוג המגע הוא לא הדוק';
             message = message.concat(looseContactErrorMessage);
         }
-
         return message;
     }
 
-    const generateErrorMessage = (emptyFields: string[], isLooseContact: boolean, validationReason: ValidationReason) => {
+    const generateErrorMessage = (emptyFields: string[], isLooseContact: boolean,
+                                  validationReason: ValidationReason, isStatusCompleted: boolean) => {
         let message = 'שים לב, ';
-        const emptyFieldsString = buildEmptyFieldsString(emptyFields, isLooseContact);
-
-        if(validationReason === ValidationReason.HANDLE_ISOLATION) return message.concat(emptyFieldsString).concat(isolationErrorMessageEnd);
-
-        return emptyFieldsString;
+        const inValidFieldsString = buildInValidFieldsString(emptyFields, isLooseContact);
+
+        switch(validationReason) {
+            case ValidationReason.HANDLE_ISOLATION:
+                return message.concat(inValidFieldsString).concat(isolationErrorMessageEnd);
+            case ValidationReason.SAVE_CONTACT: 
+                return inValidFieldsString;
+            case ValidationReason.EXCEL_LOADING: {
+                message = message.concat(inValidFieldsString);
+                if(isStatusCompleted) {
```

Comment:
Instead of having a boolean paramater `isStatusCompleted` maybe add another validationReason to the enum and have a new case